From 4f6901017ce3f2353f83998228d2c5b94041c215 Mon Sep 17 00:00:00 2001
From: hanzongcheng <hanzongcheng@huawei.com>
Date: Fri, 22 Dec 2023 14:43:45 +0800
Subject: [PATCH] disable CONFIG_WL_ROCKCHIP and CONFIG_DYNAMIC_FTRACE

* Turning on CONFIG_WL_ROCKCHIP will lead to compilation errors,
  it seems that there is a problem with the wifi driver of
  rockchip-kernel, so turn it off for now.

* The following problem occurs when building the kernel with gcc12.3
  after enabling CONFIG_DYNAMIC_FTRACE. It seems that the ftrace
  feature of the current rockchip kernel is not perfect, so turn it off.
    |
    | aarch64-openeuler-linux-ld.bfd: __patchable_function_entries has both ordered and unordered sections
    |

Signed-off-by: hanzongcheng <hanzongcheng@huawei.com>

diff --git a/arch/arm64/configs/openeuler_rk3588_defconfig b/arch/arm64/configs/openeuler_rk3588_defconfig
index e4fee1750..629a5492a 100755
--- a/arch/arm64/configs/openeuler_rk3588_defconfig
+++ b/arch/arm64/configs/openeuler_rk3588_defconfig
@@ -248,8 +248,8 @@ CONFIG_USB_RTL8150=y
 CONFIG_USB_RTL8152=y
 CONFIG_USB_USBNET=y
 CONFIG_USB_NET_QMI_WWAN=y
-CONFIG_WL_ROCKCHIP=y
-CONFIG_WIFI_BUILD_MODULE=y
+# CONFIG_WL_ROCKCHIP is not set
+# CONFIG_WIFI_BUILD_MODULE is not set
 CONFIG_AP6XXX=m
 CONFIG_BCMDHD_PCIE=y
 CONFIG_RTL8821CU=m
@@ -706,5 +706,6 @@ CONFIG_DEBUG_SPINLOCK=y
 CONFIG_DEBUG_CREDENTIALS=y
 CONFIG_RCU_CPU_STALL_TIMEOUT=60
 CONFIG_FUNCTION_TRACER=y
+# CONFIG_DYNAMIC_FTRACE is not set
 CONFIG_BLK_DEV_IO_TRACE=y
 CONFIG_LKDTM=y
-- 
2.34.1

